.github/workflows/build-container.yml

```diff
@@ -28,6 +28,17 @@ jobs:
       # now you can run kubectl to see the pods in the cluster
     - name: Try the cluster!
       run: kubectl get pods -A
+    - name: Test .plugins folder is copied in the right place
```

Comment:
I think this is not testing whether the plugins folder is getting copied, it is just copying the folder, so maybe edit the workflow name? WDYT?

Reply:
You're right, thanks.

Reply:
I moved that .plugins folder making code to where the test is done. Now the name makes sense.